src/Core/Geometry/LineSet.h

```diff
@@ -52,8 +52,8 @@ class LineSet : public Geometry3D
 	void Transform(const Eigen::Matrix4d &transformation) override;
 
 public:
-	virtual LineSet &operator+=(const LineSet &lineset);
-	virtual const LineSet operator+(const LineSet &lineset);
+	LineSet &operator+=(const LineSet &lineset);
+	const LineSet operator+(const LineSet &lineset) const;
```

Comment:
Should be ``LineSet operator+(const LineSet &lineset) const;``

Reply:
@qianyizh 
yes, you are right.